### ISSUE TITLE
MarketDataAPI is not JSON serializable

### ISSUE THREAD

**Issue by m345054** [lemon-markets/sdk-python#82]
Hello the model serialization does not work as specified in the documentation

```
client = api.create(
market_data_api_token=API_DATA,
trading_api_token=API_TRADING,
)
response = client.market_data.instruments.get()

print(response.json())
```

TypeError: Object of type MarketDataAPI is not JSON serializable

**Comment by m345054** [comment]
I have looked into the error. It is due to the _client property in instruments.py. 

```
return GetInstrumentsResponse._from_data(
            dict(data, _client=self._client, _headers=headers)
)
```

It tries to serialize the class Client from which the MarketDataAPI inherits. I think it is not important that the MarketDataAPI is serialized so an exception could be added in the JSONEncoder.

```
class JSONEncoder(json.JSONEncoder):
    def default(self, o: Any) -> Any:
        if isinstance(o, (datetime, date, time)):
            return o.isoformat()
            
        if hasattr(o, "__json__"):
            return o.__json__()

        return super().default(o)
```

The MarketDataAPI then needs a __json__ method.

```
def __json__(self):
        return
```

Here you could serialize the properties if desired. But I don't know why you should do that.

Maybe you can find a better fix.
Couldn't the _client property be removed?

**Comment by dnnsthnnr** [comment]
Thanks for reporting. The issue has been resolved in #83 and is already published as a new version 1.3.1.
